src/background/services/storage.ts

```diff
@@ -71,7 +71,7 @@ export class StorageService {
     if (tab && tab.url) {
       try {
         const tabUrl = new URL(tab.url)
-        if (tabUrl.protocol === 'https:') {
+        if (tabUrl.protocol === 'http:') {
```

Comment:
Why not both http and https?

Reply:
Just for testing purposes at the moment, since I was testing this on a local website.

Thoughts on moving the host permissions here

https://github.com/interledger/web-monetization-extension/blob/992b3460c0dce051b182f0d1a169535d2283a25d/src/shared/defines.ts#L1-L5

and define them in the webpack config based on the environment ala:

https://github.com/interledger/web-monetization-extension/blob/992b3460c0dce051b182f0d1a169535d2283a25d/webpack/dev.ts#L42-L44

Reply:
I thought I can use that for the protocol check here as well, but they are in a different format.

Reply:
Yeah this place could be better. It's in shared/constants right now, and uses process.env to switch values.

Reply:
Should we add two new defines?
- PERMISSION_HOSTS
- ALLOWED_PROTOCOLS

Reply:
Yes, better than trying to parse/convert each for different uses.

Reply:
Let me do this in a separate PR.

Reply:
I'd update plugins script to use same constants to modify manifest.json if possible.